contrib/checkbox-ce-oem/checkbox-provider-ce-oem/bin/optee_helper.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python3
+import argparse
+import glob
+import json
+import os
+import re
+import shlex
+import subprocess
+
+from look_up_xtest import look_up_app
+from pathlib import Path
+from systemd import journal
+from xtest_install_ta import find_ta_path, install_ta
+
+
+TEST_FILE_PREFIX = "optee-test-"
+
+
+def check_tee_supplicant_service():
+    try:
+        print("Looking for PID of tee-supplicant..")
+        subprocess.run(shlex.split("pgrep tee-supplicant"), check=True)
+        return True
+    except subprocess.CalledProcessError:
+        print("tee-supplicant service is not activated")
+        return False
+
```

Comment:
If you take my suggestion you may need to change here:


```suggestion
    cmd = "pgrep tee-supplicant"  
    ret = run_cmd(cmd)  
    if ret.returncode == 0:
        retrun True
    else:
        return False
```

Reply:
fixed